app/ts/inequality/Relation.ts

```diff
@@ -136,12 +161,12 @@ class Relation extends Widget {
         this.p.fill(this.color).strokeWeight(0).noStroke();
```

Comment:
The `token()` method (which this silly interface won't let me comment on) should return those symbols not in the menu by default. You may have to hard-code this list in two places and just hope that people update both; maybe note it in a comment if necessary, or else build support for ignoring the customSymbols which appear in the menu anyway and list all operators here?